trade-finance-manager-ui/server/controllers/utilisation-reports/helpers/selected-reported-fees-details-view-model-mapper.test.ts

```diff
@@ -0,0 +1,64 @@
+import { SelectedFeeRecordsDetailsResponseBody } from '../../../api-response-types';
+import { mapToSelectedReportedFeesDetailsViewModel } from '.';
+
+describe('selected-reported-fees-details-view-model-mapper', () => {
+  describe('mapToSelectedReportedFeesDetailsViewModel', () => {
+    it('should correctly map fee records to the selected reported fees details view model', () => {
```

Comment:
Can we drop the correctly from this test name please?

Reply:
Actually, is this test adding anything that the controller tests don't? I'd actually be tempted to not add unit tests here if we aren't going to test it fully or test edge cases not covered in the controller tests. Otherwise if you do want to have unit tests for this then I would move the more detailed tests from the controllers e.g. the data-sort tests to this level and then just leave the high level tests in the controllers (e.g. `should fetch and map selected fee record details`).

Reply:
Removed this in favour of the unit tests on the controller